azext_edge/edge/util/queryable.py

```diff
@@ -29,34 +24,17 @@ def __init__(self, cmd, subscriptions: Optional[List] = None):
 
         self.subscriptions = subscriptions
         self.resource_graph = ResourceGraph(cmd=cmd, subscriptions=self.subscriptions)
+        self.resource_client = get_resource_client(subscription_id=self.default_subscription_id)
+        self.console = Console()
 
     def _process_query_result(self, result: dict, first: bool = False) -> Optional[Union[dict, List[dict]]]:
         if "data" in result:
             if result["data"] and first:
                 return result["data"][0]
             return result["data"]
 
-    @property
-    def subscriptions_label(self) -> str:
-        joined_subs = "', '".join(self.subscriptions)
-        sub_label = "subscription"
-        if len(self.subscriptions) > 1:
-            sub_label += "s"
-
-        return f"{sub_label} '{joined_subs}'"
-
-    def _raise_on_resource_group_404(self, resource_group_name: str):
-        query = get_resource_group_query(resource_group_name)
-        result = self._process_query_result(result=self.resource_graph.query_resources(query=query))
-        if not result:
-            raise ResourceNotFoundError(
-                f"Resource group '{resource_group_name}' does not exist in {self.subscriptions_label}."
-            )
-
-    def query(
-        self, query: str, first: bool = False, resource_group_name: Optional[str] = None
-    ) -> Optional[Union[dict, List[dict]]]:
-        if resource_group_name:
-            self._raise_on_resource_group_404(resource_group_name)
-
+    def query(self, query: str, first: bool = False) -> Optional[Union[dict, List[dict]]]:
```

Comment:
btw pain point that I keep hitting now:
if you want to define query for a child class, it tries to override this query which isnt good practice. Thus I need to rename the command (in the child class) to run_query or something else

Reply:
noted.